README.md

```diff
@@ -0,0 +1,141 @@
+<div align="center">
+    <img src="img.png" width="20%" height="20%" alt="OneVoiceEcho-logo">
+    <h1>One Voice Echo</h1>
+     <hr>
```

Comment:
There are 2 horizontal rules. Maybe the `hr` tag is out of place

![2024-06-15_23-37](https://github.com/Firm-Collective/ai-journal/assets/65817176/19ccf80f-5aba-4ed9-922f-c4330079c1c9)

Reply:
I agree, perhaps we could also include a more detailed description about the project underneath One Voice Echo. The two tag lines seem similar and are probably redundant.

Or we could delete the duplicate One Voice Echo title and tag line and go straight to the table of contents.